### PR TITLE
this commit will create home page with all articles page usinh router…

### PR DESCRIPTION
**Pull request by sondos-ahmed**
this commit will create home page with all articles page using router and react bootstrape

### CODE REVIEW

File: src/components/Articles.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Card from "react-bootstrap/Card";
+import { useEffect, useState } from "react";
+import { Col, Row } from "react-bootstrap";
```
Comment (hevdevs) [inline src/components/Articles.jsx]: remove unused imports

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
```
Comment (hevdevs) [inline src/api.js]: remove unnecessary imports - we don't need react stuff in here

Comment (sondos-ahmed) [reply src/api.js]: done

**Review by hevdevs — CHANGES_REQUESTED** [review]
Looking good so far for ticket 4! 

Could you add some kind of loading state to signal  to the user if the data is still being fetched or not (as simple as a p tag saying 'loading' would be fine for now)? On slower connections at the moment there is just a blank page whilst the articles are still being fetched which doesn't   make for the best user experience. Please put in another PR once done :D 

Also I would advise sticking to the tickets to start with - it's great to see you doing this extra functionality for top rated articles (and I don't mean to discourage you from doing so!) but this is more supplementary functionality which makes it more difficult to review

**Review by hevdevs — CHANGES_REQUESTED** [review]
Hi Sondos - I'm getting a lot of console errors here at the moment, some from axios, some from nesting links and cannot see the loading spinner working/being shown. Could you get these errors fixed or lose the code causing them as it may be preventing the loading spinner from working? Feel free to put in an nchelp if everything is working on your end but we can't see the loading state working or run it in the emulator because of these errors

**Comment by sondos-ahmed** [comment]
I is working on my side. I have attached screen recording

[New Recording - 12_6_2022, 12_21_44 PM (1).webm](https://user-images.githubusercontent.com/17949218/205911460-b26be813-1d19-432c-ab02-a3549e574a03.webm)

**Comment by HannahBurks** [comment]
Hi Sondos, As discussed on Zoom, please add the loading to the other pages (landing page) and once done send another PR :) - nice work!

**Review by HannahBurks — APPROVED** [review]
Smashed it Sondos - feel free to now Merge! :D

**Pull request merged**